test/extract_whiteboard_test.js

```diff
@@ -0,0 +1,24 @@
+import extractWhiteboardTags from '../src/utils/extractWhiteboard';
+
+describe('extract whiteboard tags', () => {
+  it('should get the language', () => {
+    const whiteboard = '[lang=js][lang=xul]';
+    const result = extractWhiteboardTags(whiteboard);
+    expect(result).toEqual(['lang=js', 'lang=xul']);
+  });
+  it('should ignore other whiteboard tag besides language', () => {
+    const whiteboard = '[good-first-bug]';
+    const result = extractWhiteboardTags(whiteboard);
+    expect(result).toEqual([]);
+  });
+  it('should handle comma tag', () => {
```

Comment:
Have a test case for each of the following whiteboard tags:

* [lang=js][lang=xul]
* [lang=js][lang=xul][good-first-bug]
* [lang=js,xul]
* [lang=js/xul]
* [lang=js, xul] (if not too hard to integrate)

_Example:_

```js
it('should extract languages from [lang=js][lang=xul][good-first-bug]', () => {
 // ...
})
```